pilot/pkg/proxy/envoy/v1/discovery.go

```diff
@@ -481,6 +499,21 @@ func (ds *DiscoveryService) ClearCacheStats(_ *restful.Request, _ *restful.Respo
 // clearCache will clear all envoy caches. Called by service, instance and config handlers.
 // This will impact the performance, since envoy will need to recalculate.
 func (ds *DiscoveryService) clearCache() {
+	clearCacheMutex.Lock()
+	defer clearCacheMutex.Unlock()
+
+	if time.Since(lastClearCache) < time.Duration(clearCacheTime) * time.Second {
```

Comment:
greater than ?

Reply:
If the time since last clear is less than x seconds - we schedule the event. 
I think it's <